lib/snowpack/type_parser.ex

```diff
@@ -33,6 +35,10 @@ defmodule Snowpack.TypeParser do
 
   defp parse({:date, data}), do: DateTimeParser.parse_date!(data)
 
+  defp parse({:datetime, {{_year, _month, _day}, {_hour, _minute, _second}} = data}) do
+    data
+    |> NaiveDateTime.from_erl!()
```

Comment:
It is preferred style to avoid single pipes.

`NaiveDateTime.from_erl!(data)`

Reply:
sorry, should have cleaned that up, left over when i was also converting native to UTC time (leaving that to the consumer now)